src/components/InventoryGroupDetail/InventoryGroupDetail.js

```diff
@@ -70,6 +73,8 @@ const InventoryGroupDetail = ({ groupId }) => {
   }, [canViewGroup]);
 
   useEffect(() => {
+    fulfilled && data?.total === 0 && navigate('/groups');
```

Comment:
I am wondering if this can cause the page to redirect to /groups when there is a group with zero system associated with it.

Reply:
The data.total here stands for the request to GET /groups. If the list is empty, then there is no such group. Redirecting to /groups is correct in that case.